packages/transformer/src/IModelTransformer.ts

```diff
@@ -2235,6 +2232,31 @@ export class IModelTransformer extends IModelExportHandler {
       (this.targetDb as any).codeValueBehavior = "trim-unicode-whitespace";
     }
     /* eslint-enable @itwin/no-internal */
+
+    const defaultSaveTargetChanges = () => this.targetDb.saveChanges();
+    await (saveTargetChanges ?? defaultSaveTargetChanges)(this);
+    if (this.isReverseSynchronization) this.sourceDb.saveChanges();
+
+    const description = `${
+      this._isProvenanceInitTransform
+        ? `initialized branch provenance with master`
+        : this.isForwardSynchronization
+          ? "Forward sync of"
+          : "Reverse sync of"
+    } iModel: ${this.sourceDb.iModelId}`;
+
+    if (this.targetDb.isBriefcaseDb()) {
+      // FIXME<NICK> This relies on them defining an authorizationClient on iModelHost, otherwise this will fail. Not sure thats good.
+      await this.targetDb.pushChanges({
+        description,
+      });
+    }
+    if (this.isReverseSynchronization && this.sourceDb.isBriefcaseDb()) {
+      // FIXME<NICK> This relies on them defining an authorizationClient on iModelHost, otherwise this will fail. Not sure thats good.
```

Comment:
Calling attention this comment, do we care?

Reply:
If they have an authClient, they have a token no? Or are you concerned the token doesn't have push permissions or something else?

Reply:
well if they got a briefcase, they probably have an authclient with a token. if it's readonly, idk how that part works. I wouldn't care much but you can keep the comment as a non-FIXME imo